docs/book/installation/vagrant_installation.rst

```diff
@@ -0,0 +1,57 @@
+.. index::
+   single: Installation via Vagrant
+
+Sylius installation via Vagrant
+===============================
+
+.. warning::
+
+    This article assumes you're familiar with `Composer`_, a dependency manager
+    for PHP. It also assumes you have `Composer installed globally`_.
+
+What's Vagrant?
+---------------
+
+Vagrant is a tool for building complete development environments, that in case of Sylius
+will help you to quickly have full application running on your machine.
+
+.. tip::
+
+    Learn more about `Vagrant <https://www.vagrantup.com/about.html>`_.
+    Vagrant `installation <https://www.vagrantup.com/docs/installation/>`_ info.
+
+How to install Sylius using Vagrant?
+------------------------------------
+
+1. Create a new `Sylius-Standard <https://github.com/Sylius/Sylius-Standard>`_ project using composer in **no interaction mode** (``-n`` flag).
```

Comment:
`**no interaction mode**` is not mandatory. 

`Create a new `Sylius-Standard <https://github.com/Sylius/Sylius-Standard>`_ project using composer`

```
.. tip::

    Use **no interaction mode** (``-n`` flag) to have n out-of-the-box running installation. Otherwise, remember, that Vagrant shell provisioner requires proper database configuration to execute Sylius installation properly. Vagrant expects a `vagrant` as a database password.
```